package.json

```diff
@@ -30,6 +30,8 @@
   },
   "scripts": {
     "build:rtl": "rtlcss style.css style-rtl.css",
+    "build:rtl-esb": "rtlcss assets/css/editor-style-block.css assets/css/editor-style-block-rtl.css",
```

Comment:
@ntwb Would this be the correct way to create the RTL file for `assets/css/editor-style-block.css`?